### PR TITLE
Add tests for ship class

### PR DESCRIPTION
**Pull request by amikaross**

### CODE REVIEW

File: spec/ship_spec.rb [hunk 1]
```diff
@@ -1 +1,39 @@
-#Created ship_spec.rb file
+require 'rspec'
+require './lib/ship'
+
+RSpec.describe Card do
```
Comment (ajpulzone) [inline spec/ship_spec.rb]: Please update describe Card to describe Ship

File: spec/ship_spec.rb [hunk 2]
```diff
@@ -1 +1,39 @@
...
+    @cruiser = Ship.new("Cruiser", 3)
+  end
+  describe "#initialize" do 
+    it "is an instance of Ship" do
+      expect(@cruiser).to be_an_instance_of(Ship)
+    end
+
+    it "has a readable name, length, and health" do 
+      expect(@cruiser.name).to eq "Cruiser"
+      expect(@cruiser.legnth).to eq 3
```
Comment (ajpulzone) [inline spec/ship_spec.rb]: spelling correction for @cruiser.legnth (change to @cruiser.length)

**Review by ajpulzone — CHANGES_REQUESTED** [review]

File: spec/ship_spec.rb (hunk 1 above)
Comment (amikaross) [reply spec/ship_spec.rb]: Edit complete.

File: spec/ship_spec.rb (hunk 2 above)
Comment (amikaross) [reply spec/ship_spec.rb]: Edit complete

**Review by ajpulzone — APPROVED** [review]

**Pull request merged**